web/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -9,7 +9,7 @@
                 </div>
             </div>
             <div class="left">
-                <div class="search-container">
+                <div class="list-toggler-container">
```

Comment:
This div also contains the "+" button to create stream but cannot think of any other name which would be applicable here.